src/guide/essentials/forms.md

```diff
@@ -32,7 +32,7 @@ The `v-model` directive helps us simplify the above to:
 In addition, `v-model` can be used on inputs of different types, `<textarea>`, and `<select>` elements. It automatically expands to different DOM property and event pairs based on the element it is used on:
 
 - `<input>` with text types and `<textarea>` elements use `value` property and `input` event;
-- `<input type="checkbox">` and `<input type="radio">` use `checked` property and `change` event;
+- `<input type="checkbox" />` and `<input type="radio" />` use `checked` property and `change` event;
```

Comment:
I don't think these were mistakes. Writing `<input>` without a slash is correct HTML 5 usage.

Reply:
I know, it's just for consistency since we're self-closing input elements.

Reply:
@skirtles-code

Reply:
If we're going to make them consistent then I think we should consistently not include the slashes.

Reply:
@skirtles-code alright